Macros/ACS_summary_geo_source.sas

```diff
@@ -1947,6 +1962,22 @@
 					sum(C24010Ce7, C24010Ce13, C24010De7, C24010De13, C24010Ee7, C24010Ee13, 
 						C24010Fe7, C24010Fe13, C24010Ge7, C24010Ge13 );
 
+
+		mPopEmployedByInd_&_years. = C24030m1;
+			mPopEmployedAgric_&_years. = %moe_sum( var=C24030m3, C24030m30 );
```

Comment:
There should not be any commas inside the var= parameter. Example:

`mPopEmployedAgric_&_years. = %moe_sum( var=C24030m3 C24030m30 );`